app-ios/Modules/Sources/Timetable/TimetableListItemView.swift

```diff
@@ -59,8 +59,10 @@ struct TimetableListItemView: View {
                 label: {
                     if timetableItemWithFavorite.isFavorited {
                         Assets.Icons.bookmark.swiftUIImage
+                            .renderingMode(.template)
```

Comment:
[Q]
How about changing the rendering mode from asset directory since this icon is used from other places (search and bookmark screens) in the near future and should always be tinted? 👀
![image](https://github.com/DroidKaigi/conference-app-2023/assets/43767445/a4e836eb-3582-463b-acf6-4cfd13d89295)